src/js/angular/core/directives/yasgui-component/yasgui-component.directive.js

```diff
@@ -443,7 +443,6 @@ function yasguiComponentDirective(
                     .then(SparqlRestService.addKnownPrefixes)
                     .then((response) => {
                         queryString = response.data;
-                        ontotextYasguiElement.setQuery(queryString);
```

Comment:
I'm pretty sure that this line was added here on purpose. Probably it was related with the preserving the cursor position or something.

Reply:
The cursor position is preserved and remains at the end of the newly pasted line. The query is correctly set in the editor and reaches the component after every pasted line/s.